src/widgets/PageInfo.js

```diff
@@ -69,7 +72,7 @@ const renderInfoComponent = (isEditingDisabled, columnIndex, rowData, rowIndex)
       {infoString}
     </Text>
   );
-  if (rowData.onPress && !isEditingDisabled) {
+  if (rowData.onPress && !isEditingDisabled && (!isEditingDisabled && canEdit !== false)) {
```

Comment:
I don't think we need canEdit at all.
In SupplierRequisition:
```
        {
          title: `${pageInfoStrings.months_stock_required}:`,
          info: requisition.monthsToSupply,
          ...(isProgramOrder ? {} : {
          onPress: !isProgramOrder &&  this.openMonthsSelector,
          editableType: !isProgramOrder && 'selectable',
          })
        },
```

or does that look dirty ?

Reply:
It is my preferred way, my only problem is that page info configs are so big that adding this rather than the prop does make it look a bit dirty. Not sure!

Reply:
done 20821301679666ffe2b38fb80267fac54d73c1b9